docs/reference/proto-definition.md

```diff
@@ -22,9 +40,13 @@ field fieldType fieldName defaultValue
 - `fieldName` is a freely chosen name for this field.
 - `defaultValue` is a literal default value that depends on `fieldType`.
 
-Here is an example of PROTO definition:
+#### Summary
+
+Here is how a PROTO file looks like:
 
 ```
+#VRML_SIM {{ webots.version.major }} utf8
+
```

Comment:
Maybe we can add the comment example here too.

Reply:
I just wanted to keep it minimal here, given that these comments are not mandatory, unlike the header line.

Reply:
A more complete example is available later in this chapter.